build/commands/lib/config.js

```diff
@@ -285,7 +285,7 @@ Config.prototype.buildArgs = function () {
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     enable_pseudolocales: this.enable_pseudolocales,
     skip_signing: !this.shouldSign(),
-    chrome_pgo_phase: this.chromePgoPhase,
+    chrome_pgo_phase: this.isOfficialBuild() ? this.chromePgoPhase : 0,
```

Comment:
Only a non-component build can use PGO.
Otherwise you will gen compilation errors, the profiles are not designed to use with component builds.

Reply:
I think we want to explicitly set chromePgoPhase for `isOfficialBuild` and then I don't think this conditional is necessary

Reply:
in fact actually 2 is already the default for is_official_build so nothing is needed here unless we override on the command line 

```if (is_official_build &&
      # TODO(crbug.com/1052397): Remove chromeos_is_browser_only once
      # target_os switch for lacros-chrome is completed.
      (is_win || is_mac ||
       (is_linux && !chromeos_is_browser_only && !is_chromecast))) {
    chrome_pgo_phase = 2
  }```